README.md

```diff
@@ -16,28 +16,30 @@ Features:
 - Endpoints can be mocked
   - Write to it to instruct it to send back messages the same way a real device would
   - Allows separating TTY message logic from actual devices for rapid prototyping
-
-## Quickstart
-
-If you have installed serial keel (see [the install step](#running-a-server) below), please use `serial-keel help` to explore what's available.
-
-### Running a server
+car
+- Communication between server and client is done via the [WebSocket Protocol][https://datatracker.ietf.org/doc/html/rfc6455]
+  - Allows any external clients to subscribe to serial-keel server
```

Comment:
Not sure about this but feel that it should be mentioned here that the ws protocol is used, and the benefits of using that approach